rero/rero-ils-ui
# Patron : improve performance of patron account

At this time, the patron account (checkin-checkout) is displayed only
when all data has been resolved from backend ; with many linked loans
this will take a long time (more than 20 seconds before displaying
something).
This PR solves this problem : a first call is requested to get linked
item pids/barcode. Next for each barcode, we request the detail about
this item.

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

https://github.com/rero/rero-ils/issues/1246

## Dependencies

https://github.com/rero/rero-ils/pull/1262

## How to test?

- Goo to a patron account CI/CO tab (ex: Guilia) and see if all item detail are load asynchronous

## Code review check list
